Tools/Scripts/webkitpy/w3c/test_exporter.py

```diff
@@ -323,8 +336,10 @@ def make_pull_request(self):
                 _log.warning(e)
                 _log.info('Could not add label "%s" to pr #%s. User "%s" may not have permission to update labels in the %s/%s repo.' % (WEBKIT_EXPORT_PR_LABEL, pr_number, self.username, WPT_GH_ORG, WPT_GH_REPO_NAME))
         if self._bug_id and pr_number:
-            pr_url = WPT_PR_URL + str(pr_number)
-            self._bugzilla.post_comment_to_bug(self._bug_id, "Submitted web-platform-tests pull request: " + pr_url, see_also=[pr_url])
+            pr_url = f'{WPT_PR_URL}{pr_number}'
+            # TODO: Waiting to land PR #45278 to call this function
+            # self._bug.add_related_links([pr_url])
```

Comment:
This is a blocker, given https://github.com/web-platform-tests/wpt-pr-bot/blob/f03de165700a8dcb8ab47b0a15c8c0be9f083fe2/lib/metadata/webkit.js#L11-L13 verifies the link back here.

Reply:
Landed! I added to the unit tests to check for this as well